.github/workflows/ci.yml

```diff
@@ -19,62 +19,128 @@ jobs:
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with Symfony ^5'
-            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
+            job-description: 'Fixer'
+            run-fixer: 'yes'
+            run-phpdoc-to-native-type: 'yes' # should be checked on the lowest supported PHP version
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
+            job-description: 'tests'
+            run-tests: 'yes'
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.0'
+            job-description: 'Fixer'
+            run-fixer: 'yes'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.0'
+            job-description: 'tests'
+            run-tests: 'yes'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.1'
-            job-description: 'with Symfony ^6'
+            job-description: 'Fixer with Symfony ^6'
+            run-fixer: 'yes'
             execute-flex-with-symfony-version: '^6' # explicit check for Symfony 6.x compatibility
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.1'
+            job-description: 'tests with Symfony ^6'
+            run-tests: 'yes'
+            execute-flex-with-symfony-version: '^6' # explicit check for Symfony 6.x compatibility
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.2'
+            job-description: 'Fixer'
+            run-fixer: 'yes'
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.2'
+            job-description: 'Fixer with migration rules'
+            run-fixer: 'yes'
+            run-migration-rules: 'yes' # should be checked on the highest supported PHP version
+
           - operating-system: 'ubuntu-20.04'
             php-version: '8.2'
-            job-description: 'with migration rules'
-            migration-rules: 'yes' # should be checked on the highest supported PHP version
+            job-description: 'tests with migration rules'
+            run-tests: 'yes'
+            run-migration-rules: 'yes' # should be checked on the highest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.2'
             job-description: 'auto-review'
+            run-tests: 'yes'
             phpunit-flags: '--group auto-review'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.2'
-            job-description: 'with calculating code coverage'
-            calculate-code-coverage: 'yes'
+            job-description: 'tests'
+            run-tests: 'yes'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.2'
-            job-description: 'with deployment'
-            execute-deployment: 'yes'
+            job-description: 'code coverage'
+            code-coverage: 'yes'
```

Comment:
everywhere else you have `run-*`, here, there is no `run-*`, and setting up `code-coverage` is marking the job to run tests as well. I was rather expecting to see `run-tests && code-coverage` or `run-tests || run-tests-with-code-coverage`.

I got tricked [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7366/files#r1380415388)

Reply:
pls consider to apply this in new PR

Reply:
I believe adding `collect-` to this flag is enough, because it's obvious that you can't collect code coverage without running tests, so it's implicit. And since this is not the same step as regular tests, then I don't see the need to combine 2 flags for that.